integration-libs/cdc/manage-users/manage-users-button/cdc-list.module.ts

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
change the enclosing folder name from `manage-users-button` to `manage-users-list`